sui/src/config.rs

```diff
@@ -69,6 +70,15 @@ impl Clone for AuthorityPrivateInfo {
     }
 }
 
+fn socket_addr_from_hostport(host: &str, port: u16) -> SocketAddr {
+    let mut addresses = format!("{host}:{port}")
+        .to_socket_addrs()
```

Comment:
i'm probably being too paranoid, but adding panicking and blocking functions for things that might actually fail (e.g. dns resolution) is scary. i know its only being called at start up, but you could imagine someone moving this to some other location and using it unsafely.

a short warning comment would make me happy 😄

Reply:
@mystenmark ok comment added, thanks